src/css/jwplayer/flags/header.less

```diff
@@ -3,7 +3,7 @@
     overflow: visible;
 }
 
-.jw-header {
+.jw-top {
```

Comment:
We still want to call it `.jw-flag-header`, or should we `.jw-flag-top`? Also, file name still has header.